core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/StudyImpl.java

```diff
@@ -344,4 +339,16 @@ public ChangeLog createChangeLog(String summary, String columnsChanged, User use
     changeLog.setUser(user);
     return changeLog;
   }
+
+  @Override
+  public String getDeleteType() {
+    return "Study";
+  }
+
+  @Override
+  public String getDeleteDescription() {
+    Project project = getProject();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "project" which hides the field declared at line 68. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)